StudyPathTraker/StudyPathTraker/Collection/SPCategoriesCollectionViewFlowDelegate.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  SPCategoriesCollectionViewFlowDelegate.swift
+//  StudyPathTraker
+//
+//  Created by Rafael Lopez on 4/30/18.
+//  Copyright © 2018 Jerti. All rights reserved.
+//
+
+import UIKit
+
+
+class SPCategoriesCollectionViewFlowDelegate: NSObject, UICollectionViewDelegateFlowLayout {
+    
+    private var categoryEdgeInsets: UIEdgeInsets
```

Comment:
this will look better  if you make a `struct` called `CategoriesFlowConfiguration` and put there this values for configuration.

Reply:
Ready